Make Layout container validation fail on non existent fields (#2927)
demisto/demisto-sdk: CHANGELOG.md

* Added **generate-unit-tests** documentation to the repo README.
* Added the `hiddenpassword` field to the integration schema, allowing **validate** to run on integrations with username-only inputs.
* Fixed an issue where **doc-review** required dot suffixes in release notes describing new content.

> * Fixed an issue where **validate** passed a layouts container with incident fields not present in `id_set.json` even when `--post-commit` was indicated.

Reviewer: ```suggestion
* Fixed an issue where **validate** didn't recognize layouts with incident fields missing from `id_set.json` even when `--post-commit` was indicated.
```
